app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,86 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * 유저를 관리합니다.
+ */
+@Service
+public class UserService {
+
+    private UserRepository userRepository;
+
+    @Autowired
+    UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * 유저 목록을 반환합니다.
+     *
+     * @return 유저 목록
+     */
+    List<User> findAllUsers() {
+        return userRepository.findAll();
+    }
+
+    /**
+     * 주어진 id와 일치하는 유저를 반환합니다.
+     *
+     * @param id 유저 id
+     * @return 주어진 id와 일치하는 유저
+     * @throws '유저를 찾지 못했다'는 예외
```

Comment:
`@throws`에는 예외 클래스 이름도 적어주셔야 해요. JavaDoc이 빌드되면 코드 본문은 보이지 않거든요. 여기에 지정된 예외 클래스 이름은 IDE의 리팩토링 도구를 사용하면 클래스 이름을 바꿀 때 같이 수정됩니다.

```suggestion
     * @throws UserNotFoundException '유저를 찾지 못했다'는 예외
```